<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiosDialog</class>
 <widget class="QDialog" name="RadiosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>661</width>
    <height>428</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>661</width>
    <height>428</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Available Radios</string>
  </property>
  <widget class="QTableWidget" name="radioListTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Channel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rcvr</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Xmtr</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Running</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="radioStartButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="radioCancelButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Receive Audio</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QRadioButton" name="radioAudioLocalChk">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Local</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioAudioRemoteChk">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remote</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>320</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mic Audio</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QRadioButton" name="radioMicLocalChk">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Local</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioMicRemoteChk">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remote</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>220</y>
     <width>101</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Sample Rate</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QRadioButton" name="radio48000Chk">
      <property name="text">
       <string>48000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio96000Chk">
      <property name="text">
       <string>96000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio192000Chk">
      <property name="text">
       <string>192000</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio384000Chk">
      <property name="text">
       <string>384000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio768000Chk">
      <property name="text">
       <string>768000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio1536000Chk">
      <property name="text">
       <string>1536000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioCancelButton</sender>
   <signal>released()</signal>
   <receiver>RadiosDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
